src/sentry/static/sentry/app/views/releases/detail/organization/releaseAllEvents.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import SentryTypes from 'app/sentryTypes';
+import GroupList from 'app/components/groupList';
+import {t} from 'app/locale';
+
+export default class ReleaseAllEvents extends React.Component {
+  static contextTypes = {
+    release: SentryTypes.Release,
+  };
+
+  render() {
+    let {orgId} = this.props.params;
+    return (
+      <div>
+        <div className="alert alert-block">
+          <Link
+            to={{
+              pathname: `/organizations/${orgId}/issues/`,
+              query: {query: 'release:' + this.context.release.version},
+            }}
+          >
+            <span className="icon icon-open" />
+            {t('View all events seen in this release in the stream')}
```

Comment:
Should this be `issues` and not `events`?

Reply:
This is the same as the current text, will update though since issues makes more sense